ocaml/xapi/storage_access.ml

```diff
@@ -454,9 +454,11 @@ module SMAPIv1 = struct
                      extra = attach_info_v1.Smint.xenstore_data;
                      backend_type = "vbd3"
                    };
-                   (* Currently we always get a BlockDevice from SMAPIv1, never a File, not even for ISOs *)
```

Comment:
perhaps you could amend this comment rather than deleting it to explain what has changed

Reply:
done